tests/tripadvisor2mimir/mod.rs

```diff
@@ -0,0 +1,172 @@
+use std::iter;
```

Comment:
Thanks for the test!

And sorry I've procrastinated on them, I'll rebase my branch and add photo-related tests as soon as this one is stabilized.

Reply:
Don't worry, I really appreciate writing tests :)